tests/yarn_files/conditions.json

```diff
@@ -89,5 +89,55 @@
 			"y": 252
 		},
 		"colorID": 0
+	},
+	{
+		"title": "TextAfterOption",
+		"tags": "Tag",
+		"body": "Text before\n[[FinalOption|Objective]]\nText after",
```

Comment:
so in this new fix, Text after will get rendered, then the option will appear?

Reply:
Yes and test_runner.js **'Can handle text after an option'**  demonstrates this.